crates/solidity/testing/sourcify/src/run.rs

```diff
@@ -271,6 +468,18 @@ impl Diagnostic for BindingError {
                     symbol = cursor.node().unparse()
                 )
             }
+            Self::MissingDefinition(cursor) => {
```

Comment:
not blocking: I wonder if we should split the now 500+ lines `run.rs` into a `run/` module with individual test runners? since now we have at least 4 individual checks that can be enabled in a given run.